modules/ROOT/pages/download/nb26/index.adoc

```diff
@@ -0,0 +1,139 @@
+:url-download: https://www.apache.org/dyn/closer.lua/netbeans/
+:url-download-keychecksum: https://downloads.apache.org/netbeans/
+// :url-download: https://archive.apache.org/dist/netbeans/
+// :url-download-keychecksum: https://archive.apache.org/dist/netbeans/
+
+//// 
+url-download depends of release status archived or not
+https://archive.apache.org/dist/netbeans/  (//archived)
+https://www.apache.org/dyn/closer.lua/netbeans/ (//current)
+
+url-download-keychecksum depends of release status archived or not
+https://archive.apache.org/dist/netbeans/  (//archived)
+https://downloads.apache.org/netbeans/ (//current)
+////
+
+Apache NetBeans {netbeans-version} was released on May xxx, 2025.
+// TODO update date                                ^^^^
+
+////
+NOTE: It's mandatory to link to the source. It's optional to link to the binaries.
+NOTE: It's mandatory to link against https://www.apache.org for the sums & keys. https is recommended.
+NOTE: It's NOT recommended to link to github.
+////
+
+*Binaries (Platform Independent):*
+
+* link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip?action=download[netbeans-{netbeans-version}-bin.zip]
+ (link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip.sha512[SHA-512],
+ link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-bin.zip.asc[PGP ASC])
+
+*Source:*
+
+* link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip?action=download[netbeans-{netbeans-version}-source.zip] 
+(link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.sha512[SHA-512], 
+link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.asc[PGP ASC])
+
+Officially, it is important that you link:https://www.apache.org/dyn/closer.lua#verify[verify the integrity]
+of the downloaded files using the PGP signatures (.asc file) or a hash (.sha512 files).
+The PGP keys used to sign this release are available link:https://downloads.apache.org/netbeans/KEYS[here].
+
+*Release Notes:*
+
+* link:https://github.com/apache/netbeans/releases/tag/{netbeans-version}[Github Link]
+
+== Community Installers
+
+- link:https://installers.friendsofapachenetbeans.org/[Friends of Apache NetBeans] - Windows, macOS and
+Linux (.deb / .rpm / .AppImage) built with link:https://github.com/apache/netbeans-nbpackage/[NBPackage]. Most
+include a local JDK runtime for the IDE to run on, for a self-contained out-of-the-box experience.
+- link:https://snapcraft.io/netbeans[Linux snap package]
+
+IMPORTANT: Individual NetBeans committers may provide additional binary packages as a convenience.
+While built using the Apache NetBeans release, they are not releases of the Apache Software
+Foundation. They may include other contents (eg. JDK) under additional license terms.
```

Comment:
IMPORTANT: These installers and packages are provided by various NetBeans committers as a convenience. While built using the Apache NetBeans release, they are not releases of the Apache Software Foundation. They may include other contents (eg. JDK) under additional license terms.